project/AppDependencies.scala

```diff
@@ -4,22 +4,21 @@ import play.core.PlayVersion
 
 object AppDependencies {
   
-  private val playVersion = "play-26"
+  private val playVersion = "play-27"
```

Comment:
You are configuring everything to be play27 here, you need to modify the dependency based on the play version, see here for an example: 
https://github.com/hmrc/auth-client/blob/97203bc0820ffda7a6372defac7f0687523190b7/project/BuildDependencies.scala#L6